OMEdit/OMEditLIB/OMS/OMSSimulationOutputWidget.cpp

```diff
@@ -91,28 +91,33 @@ OMSSimulationOutputWidget::OMSSimulationOutputWidget(const QString &cref, QWidge
     mResultFileLastModifiedDateTime = QDateTime::currentDateTime();
   }
   mIsSimulationRunning = false;
-  // initialize the model
-  if (OMSProxy::instance()->initialize(mCref)) {
-    // start the asynchronous simulation
-    qRegisterMetaType<oms_status_enu_t>("oms_status_enu_t");
-    connect(this, SIGNAL(sendSimulationProgress(QString,double,oms_status_enu_t)), SLOT(simulationProgress(QString,double,oms_status_enu_t)));
-    if (OMSProxy::instance()->simulate_asynchronous(mCref)) {
-      mIsSimulationRunning = true;
-      mpCancelSimulationButton->setEnabled(true);
+  // instantiate the model
+  if (OMSProxy::instance()->instantiate(mCref)) {
+    // initialize the model
+    if (OMSProxy::instance()->initialize(mCref)) {
+      // start the asynchronous simulation
+      qRegisterMetaType<oms_status_enu_t>("oms_status_enu_t");
+      connect(this, SIGNAL(sendSimulationProgress(QString,double,oms_status_enu_t)), SLOT(simulationProgress(QString,double,oms_status_enu_t)));
+      if (OMSProxy::instance()->simulate_asynchronous(mCref)) {
+        mIsSimulationRunning = true;
+        mpCancelSimulationButton->setEnabled(true);
+      } else {
+        mpProgressLabel->setText(tr("Simulation using the <b>%1</b> model is failed. %2").arg(mCref).arg(GUIMessages::getMessage(GUIMessages::CHECK_MESSAGES_BROWSER)));
+        mpProgressBar->setValue(mpProgressBar->maximum());
+        mpArchivedOMSSimulationItem->setStatus(tr("Simulation failed!"));
+        OMSProxy::instance()->terminate(mCref);
+      }
     } else {
-      mpProgressLabel->setText(tr("Simulation using the <b>%1</b> model is failed. %2").arg(mCref).arg(GUIMessages::getMessage(GUIMessages::CHECK_MESSAGES_BROWSER)));
+      mpProgressLabel->setText(tr("Initialization using the <b>%1</b> model is failed. %2").arg(mCref).arg(GUIMessages::getMessage(GUIMessages::CHECK_MESSAGES_BROWSER)));
```

Comment:
Sounds weird, maybe: "Initialization of the model <b>%1</b> has failed: %2"

Reply:
Isn't `Initialization of the model %1 is failed: %2` is more appropriate? Then we have sentences like `Simulation using the <b>%1</b> model is cancelled.` and `Simulation using the <b>%1</b> model is finished.` which should also be updated like `Simulation of the model <b>%1</b> is cancelled.` and `Simulation of the model <b>%1</b> is finished.`

Reply:
Now is worse :)
These are actions performed on the model so not "using" and not "is", the action has been done it has failed or worked so we need to use "has".

I would go for "Initialization/Simulation of model X has failed. msg"

Reply:
Ah, i now saw you changed using to of the model, so is better. I guess I should stop looking at PRs Friday night :)
Also "Simulation using the <b>%1</b> model is cancelled." is not ok, I would go for "Simulation of model <b>%1</b> has been cancelled."

But I'm no expert in English so maybe we should ask somebody else, @sjoelund any comments?